app/models/dataset_record/darwin_core/occurrence.rb

```diff
@@ -428,6 +458,7 @@ def parse_record_level_class
 
     # basisOfRecord: [Check it is 'PreservedSpecimen', 'FossilSpecimen']
     basis = get_field_value(:basisOfRecord)
+    basis = basis.titleize.gsub(' ', '') if basis.include? '_' # Reformat GBIF occurrence download basis of records (e.g., PRESERVED_SPECIMEN to PreservedSpecimen)
```

Comment:
We may need to isolate (re)formatter methods like this in an agnostic lib/utilities/<lib>.rb.